build.sbt

```diff
@@ -155,7 +155,7 @@ lazy val flink116ModelCompat = (project in file("flink116/model"))
     dependencyOverrides ++= Seq(
       "org.apache.kafka" % "kafka-clients" % kafkaV,
       "org.apache.kafka" %% "kafka" % kafkaV,
-    ),
+    ) ++ flinkOverrides(flink116V),
```

Comment:
previously, without overrides these tests were passing without patched class because they were running against flink-core 1.19.1 - with override they run against 1.16 and fail without patch as expected